### PR TITLE
Refactor code to add prefixes to variable names use camel case convention.

### LINKED ISSUE #17: Follow a specific naming convention for variables and classes
like parameters having p_name, member variables having m_name, etc.

### PR DESCRIPTION
**Pull request by shubham4019**
Closes #17

### CODE REVIEW

File: ProcSDF/GUI/Inspector.h [hunk 1]
```diff
@@ -18,12 +18,12 @@ class Inspector
...
 
-	void append_to_same_line_if_applicable(float space_needed)
+	void appendToSameLineIfApplicable(float p_spaceNeeded)
 	{
 		ImGui::SameLine();
-		if (ImGui::GetContentRegionAvail().x <= space_needed)
+		if (ImGui::GetContentRegionAvail().x <= p_spaceNeeded)
 		{
 			ImGui::NewLine();
 		}
```
Comment (angad-k) [inline ProcSDF/GUI/Inspector.h:29]: Note to self, move this to GUIutilities

File: ProcSDF/Main.cpp [hunk 2]
```diff
@@ -13,42 +13,42 @@ extern "C" {
...
 {
 	//GUI needs to be created before renderer since it also sets up OpenGL
-	GUI* gui = GUI::get_singleton();
-	Renderer* renderer = Renderer::get_singleton();
-	ShaderGenerator::get_singleton();
-	NodeGraph::get_singleton();
-	Inspector::get_singleton();
-	NodeEditor::get_singleton();
+	GUI* l_gui = GUI::getSingleton();
+	Renderer* m_renderer = Renderer::getSingleton();
```
Comment (angad-k) [inline ProcSDF/Main.cpp:27]: not `l_` ?

File: ProcSDF/Rendering/Renderer.h [hunk 3]
```diff
@@ -7,71 +7,71 @@
...
-	float cachedHeight = 600;
+	float m_cachedWidth = 600;
+	float m_cachedHeight = 600;
 
-	int cameraOrigin;
-	int focalLength;
-	int viewportSize;
-	int lightPosition;
+	int l_cameraOrigin;
+	int l_focalLength;
```
Comment (angad-k) [inline ProcSDF/Rendering/Renderer.h:20]: `m?`

File: ProcSDF/Rendering/Renderer.h [hunk 4]
```diff
@@ -7,71 +7,71 @@
...
+	int l_viewportSize;
```
Comment (angad-k) [inline ProcSDF/Rendering/Renderer.h:21]: m_ ?

File: ProcSDF/Rendering/Renderer.h [hunk 5]
```diff
@@ -7,71 +7,71 @@
...
+	int m_lightPosition;
 
-	float cameraOriginValue[3] = { 0.0, 0.0, 5.0 };
-	float focalLengthValue = 1.0;
+	float l_cameraOriginValue[3] = { 0.0, 0.0, 5.0 };
+	float l_focalLengthValue = 1.0;
```
Comment (angad-k) [inline ProcSDF/Rendering/Renderer.h:25]: m_ ?

File: ProcSDF/Rendering/ShaderGenerator.cpp [hunk 6]
```diff
@@ -7,309 +7,309 @@
...
-	file_content.append((std::istreambuf_iterator< char >(sourceFile)), std::istreambuf_iterator< char >());
-	file_content.append("\n");
-	return file_content;
+std::string ShaderGenerator::fetchFileContent(std::string p_fileName) {
+	std::string l_fileContent = "\n";
+	std::string l_filePath = sdf::SHADER_FOLDER_PATH + p_fileName + sdf::SHADER_FILE_EXTENSION;
+	std::ifstream sourceFile(l_filePath);
+	l_fileContent.append((std::istreambuf_iterator< char >(sourceFile)), std::istreambuf_iterator< char >());
+	l_fileContent.append("\n");
+	return l_fileContent;
```
Comment (angad-k) [inline ProcSDF/Rendering/ShaderGenerator.cpp:77]: note to self : move to this an os file while writing the custom nodes PR

File: ProcSDF/Rendering/ShaderGenerator.cpp [hunk 7]
```diff
@@ -7,309 +7,309 @@
...
+		int l_index = 0;
 
-		if (nd->is_object_node) {
-			function_name = "object";
-			function_name.append("_");
-			function_name.append(std::to_string(ShaderGenerator::node_id_to_object_id_map[nd->id]));
+		if (l_nd->m_isObjectNode) {
+			l_functionName = "object";
+			l_functionName.append("_");
+			l_functionName.append(std::to_string(ShaderGenerator::m_nodeIDToObjectIDMap[l_nd->m_ID]));
```
Comment (angad-k) [inline ProcSDF/Rendering/ShaderGenerator.cpp:204]: nodeIDToObjectIDMap only please

Comment (angad-k) [reply ProcSDF/Rendering/ShaderGenerator.cpp:204]: nevermind

**Review by angad-k — COMMENTED** [review]
Will merge after fixing the minor issues

File: ProcSDF/Rendering/ShaderGenerator.cpp (hunk 7 above)
Comment (shubham4019) [reply ProcSDF/Rendering/ShaderGenerator.cpp:204]: okay

**Pull request merged**